### DISCUSSION TITLE
Проблема в Супергруппах

### DISCUSSION THREAD

**Discussion by sdmway** [aiogram/aiogram#1481] [Q&A]
Привет. Недавно столкнулся с проблемой в супергруппах, а именно в темах.

Раньше для команд у меня стояла проверка:

```python
if message.reply_to_message:
    user_id = message.reply_to_message.from_user.id
else:
    args = message.text.split(" ")
    user_id = args[1]
```
Но после добавления тем, при любом использовании команды в теме я получаю свой же айди, потому что оно считает, что я постоянно отвечаю на свое сообщение. Для решения этой проблемы мне понадобилось написать мини-решение:

```python
if message.reply_to_message.from_user.id == message.from_user.id:
    args = message.text.split(" ")
    user_id = args[1]
else:
    user_id = message.reply_to_message.from_user.id
```
Выглядит не очень. Так вот, мой вопрос: есть ли какое-то решение для этого или легкий способ проверки? Просто странно, что каждое сообщение считает как `reply_to_message`.

**Comment by dmitryshm** [comment]
Здесь похоже на то, что какой-то ущемлённый хохол в авторах типа блокирует всю активность на русском. Группа в тг тому в подтверждение. Думается, когда 404 скоро превратится в нормальную санитарную зону - такой проблемы не будет.

**Comment by JrooTJunior** [comment]
All messages in the threads is replies to the topic start message. This is exactly it works on the Telegram platform, so, to work with replies in the threads you should check it.

The Bot API documentation says: https://core.telegram.org/bots/api#message

ield | Type | Description
-- | -- | --
... | ... | ...
reply_to_message | Message | Optional. For replies in the same chat and message thread, the original message. Note that the Message object in this field will not contain further reply_to_message fields even if it itself is a reply.
... | ... | ...

Condition `if message.reply_to_message.from_user.id == message.from_user.id:` is not correct for the topic, you should check 
`forum_topic_created` attribute in the `reply_to_message` instead.
